diff --git a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt
index 51338df..f35459c 100644
--- a/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt
+++ b/node_modules/react-native-webview/android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt
@@ -341,7 +341,7 @@ class RNCWebViewManagerImpl(private val newArch: Boolean = false) {
         }
         "injectJavaScript" -> webView.evaluateJavascriptWithFallback(args.getString(0))
         "loadUrl" -> {
-          val url = args?.getString(0) ?: throw RuntimeException("Arguments for loading an url are null!")
+          val url = args?.getString(0) ?: ""
           webView.progressChangedFilter.setWaitingForCommandLoadUrl(false)
           webView.loadUrl(url)
         }
